.azurepipelines/BaseTools-Build-For-Publication.yml

```diff
@@ -0,0 +1,146 @@
+name: BaseTools_Bin_$(release_version)_$(Rev:r)
+
+variables:
+  linux_build_output_path: '$(Build.Repository.LocalPath)/BaseTools/Source/C/bin'
+  linux_artifact_name: 'LinuxOutput'
+  windows_build_output_path: '$(Build.Repository.LocalPath)\BaseTools\Bin\Win32'
+  windows_artifact_name: 'WindowsOutput'
+  temp_publication_directory: 'PackageStaging'
+  package_artifact_name: 'NugetReleasePackage'
+
+jobs:
+  - job: linux_build
+    displayName: Linux Build
+    pool:
+      vmImage: ubuntu-latest
+
+    workspace:
+      clean: all
+
+    steps:
+    - checkout: self
+      clean: true
+  
+    - task: UsePythonVersion@0
+      inputs:
+        versionSpec: '3.7.x'
+        architecture: 'x64'
+      displayName: Install correct Python
+
+    - script: pip install -r requirements.txt --upgrade
+      displayName: 'Install/Upgrade pip modules'
+
+    # Set default
+    - bash: |
+        echo "##vso[task.setvariable variable=pkg_count]${{ 1 }}"
+
+    - task: CmdLine@1
+      displayName: Update Environment
+      inputs:
+        filename: stuart_update
+        arguments: -c .pytool/CISettings.py
+
+    - template: templates/basetools-build-steps.yml
+      parameters:
+        tool_chain_tag: 'GCC5'
+
+    - powershell: Get-ChildItem -Recurse -Include "*_path_env.yaml", "*_path_env.yml" -Path $(linux_build_output_path) | Remove-Item
```

Comment:
i think it would be easier to do the build step with a parameter that said don't output yaml file then you don't need to delete

Reply:
Oooo... good note.